src/test/java/seedu/address/ui/PersonListPanelTest.java

```diff
@@ -103,7 +103,7 @@ private Path createXmlFileWithPersons(int personCount) throws Exception {
         }
         builder.append("</addressbook>\n");
 
-        Path manyPersonsFile = Paths.get(TEST_DATA_FOLDER + "manyPersons.xml");
+        Path manyPersonsFile = TEST_DATA_FOLDER.resolve("manyPersons.xml");
```

Comment:
Please split this out into another PR because we are not the original author of the fix and it is not good to mix our changes with upstream changes.